src/components/App.js

```diff
@@ -19,20 +19,28 @@ const App = () => {
   const [selectFilter, setSelectFilter] = useState("all");
   const [isShowCheckbox, setShowCheckbox] = useState(false);
   const [completedAll, setCompletedAll] = useState(false);
+  const [messageError, setMessageError] = useState("");
 
-  const addTask = () => {
+  const addTask = async () => {
     if (value) {
-      setItems([
-        ...items,
-        {
-          id: Date.now(),
+      try {
+        const response = await addTodoToDB({
           value: value,
           completed: false,
-        },
-      ]);
-
-      setValue("");
-      setCompletedAll(false);
+        });
+
+        if (response === undefined)
+          throw new Error(
```

Comment:
You can remove try catch from API functions and catching it here. You have already try catch here